content/en/tracing/advanced/adding_metadata_to_spans.md

```diff
@@ -335,9 +335,17 @@ const tracer = require('dd-trace').init({
 {{% /tab %}}
 {{% tab ".NET" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to learn more.
+Use the environment variable `DD_TRACE_GLOBAL_TAGS` to add [tags][1] to all the generated [spans][2]. See the [.NET configuration][3]
```

Comment:
The wording in here should indicate that this setting can be set in other ways aside from an environment variable.

Reply:
Ok, sure.